main.go

```diff
@@ -78,12 +73,11 @@ func main() {
 		case <-reload.C:
 			res.Reload()
 		case masters := <-changed:
-			timeout.Stop()
 			logging.VeryVerbose.Printf("new masters detected: %v", masters)
 			res.SetMasters(masters)
 			res.Reload()
 		case err := <-errch:
-			logging.Error.Fatal(err)
+			log.Fatal(err)
```

Comment:
ugh yes. was keeping it in line with line 59 and 61. Want this changed as well to use the Error logger?